ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/views/RelevantPurchasesListView.kt

```diff
@@ -33,6 +33,7 @@ internal fun RelevantPurchasesListView(
     virtualCurrencies: VirtualCurrencies?,
     appearance: CustomerCenterConfigData.Appearance,
     localization: CustomerCenterConfigData.Localization,
+    supportTicket: CustomerCenterConfigData.Support.SupportTickets?,
```

Comment:
Should this be plural? `supportTickets`. Or maybe the type should be renamed to `SupportTicket`...